pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceTest.java

```diff
@@ -593,8 +593,6 @@ public void testTlsEnabledWithoutNonTlsServicePorts() throws Exception {
 
         } catch (Exception e) {
             fail("should not fail");
-        } finally {
```

Comment:
why delete this code?

Reply:
If close the `pulsarClient`, it will cause other test methods can't use the `pulsarClient` and there is a clean operation after the test.